src/os/ui/ol/interaction/dragboxinteraction.js

```diff
@@ -93,14 +91,13 @@ os.ui.ol.interaction.DragBox.prototype.update = function(mapBrowserEvent) {
       this.extent = [];
     }
 
-    var start = ol.proj.toLonLat(this.startCoord, this.getMap().getView().getProjection());
-    var end = ol.proj.toLonLat(this.endCoord, this.getMap().getView().getProjection());
+    if (this.startCoord && this.endCoord) {
+      this.extent[0] = Math.min(this.startCoord[0], this.endCoord[0]);
```

Comment:
`update2D` expects the extent to be in lon/lat, so drag boxes are only working in 4326.